internal/agent/device/status/resources.go

```diff
@@ -0,0 +1,52 @@
+package status
+
+import (
+	"context"
+
+	"github.com/flightctl/flightctl/api/v1alpha1"
+	"github.com/flightctl/flightctl/internal/agent/device/resource"
+	"github.com/flightctl/flightctl/pkg/log"
+)
+
+var _ Exporter = (*Resources)(nil)
+
+type Resources struct {
+	manager resource.Manager
+	log     *log.PrefixLogger
+}
+
+func newResources(log *log.PrefixLogger, manager resource.Manager) *Resources {
+	return &Resources{
+		manager: manager,
+		log:     log,
+	}
+}
+
+func (r *Resources) Export(ctx context.Context, status *v1alpha1.DeviceStatus) error {
+	resource := r.manager.Usage()
+	diskStatus, err := diskUsageStatus(resource.DiskUsage)
+	if err != nil {
+		// TODO: add error to status
```

Comment:
This is MVP and now sets the disk monitor to error status. Will go over options with UI in next design sync.